package.json

```diff
@@ -1,33 +1,45 @@
 {
-  "name": "@parity/dapp-addresses",
-  "description": "Parity default addressbook view",
-  "version": "1.99.99",
+  "name": "bond-dapp-addresses",
```

Comment:
Keep the package name. We install this (compiled version) when building the UI - as it stands it won't quite work since the names are not a match. This is the same dapp, just the next version.